config/.js/src/main/scala/bloop/config/PlatformFiles.scala

```diff
@@ -4,6 +4,7 @@ object PlatformFiles {
   type Path = String
   val emptyPath: Path = getPath("")
   def getPath(path: String): Path = path
+  def getFileName(path: Path): Path = path.split('/').last
```

Comment:
Won't this break on windows?

Reply:
Not sure, but we don't actually use it, since the method is only used in integration tools for build tools.

I can modify it to use the system separator just in case 🤔

Reply:
> but we don't actually use it

meh, don't even worry about it then.